site/components/pages/Buy/index.tsx

```diff
@@ -175,9 +175,9 @@ export const Buy: NextPage<Props> = ({}) => {
               traditional currencies - powered by our partners at Mobilum.
               Coming Soon!
             </Text>
-            <button disabled={true} className={styles.comingSoonButton}>
-              Coming Soon
-            </button>
+            <a href={"https://dapp.klimadao.finance/#/buy"}>
```

Comment:
This can go in `urls` object in `lib/constants.ts`
maybe call it `urls.buy_dapp`?